### ISSUE TITLE
can you launch a program rather than xscreensaver?

### ISSUE THREAD

**Issue by acme64** [google/xsecurelock#137]
can i do something like have the screensaver be _htop_ or some other program? in a similar way you can call other programs within xscreensaver-phosphor with _-program_

**Comment by divVerent** [comment]
Not quite, however, you should be able to use xscreensaver-phosphor from
xsecurelock to render the output.
